### ISSUE TITLE
Chapter dashboard: "See on website" button does not comply with WCAG 1.4.3

### ISSUE THREAD

**Issue by agriyakhetarpal** [fossunited/fossunited#1207]
Hi! When managing a chapter through the dashboard, I see this button:

<img width="189" height="132" alt="Image" src="https://github.com/user-attachments/assets/1da945c9-caed-48f4-9a0c-9fd39129525a" />

<hr>

The text here does not comply with "Contrast (Minimum) - Level AA". The success criterion for this accessibility standard requires at least a 4.5:1 contrast ratio for normal text, and at least a 3:1 contrast ratio for large text.

For your reference: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

I would be happy to take this on and provide a fix if y'all are interested.

Thank you!

**Comment by idlip** [comment]
Ah. Nice catch. Thanks for noticing it and making issue

I wonder all this why nobody observed it... (even I did not lol)

**Comment by idlip** [comment]
Fixed! Should be deployed by today or tomorrow!

Please do open more if found any such issues!
